internal/backends/winit/winitwindowadapter.rs

```diff
@@ -203,11 +203,6 @@ impl WinitWindowAdapter {
         let mut window_builder =
             winit::window::WindowBuilder::new().with_transparent(true).with_visible(false);
 
-        if std::env::var("SLINT_FULLSCREEN").is_ok() {
```

Comment:
I'm thinking we should keep the env variable for now in addition, or that'd be a breaking change.

Reply:
Done.